app/src/main/kotlin/org/equeim/tremotesf/ui/torrentslistfragment/TorrentsAdapter.kt

```diff
@@ -125,7 +127,16 @@ class TorrentsAdapter(private val fragment: TorrentsListFragment) :
                 }
                 etaTextView.text = FormatUtils.formatDuration(context, torrent.eta)
 
-                progressBar.progress = (torrent.percentDone * 100).toInt()
+                val progress = (torrent.percentDone * 100).toInt()
+                progressBar.progress = progress
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                    progressBar.progressTintList = if (progress == 100) {
+                        ColorStateList.valueOf(context.getColor(R.color.color_downloaded_torrents))
+                    } else {
+                        ColorStateList.valueOf(context.getColor(R.color.color_primary))
```

Comment:
Or maybe not....

Reply:
It seems that Android sets default tint with primary color so we indeed need to restore it.
Anyway, we need to get the color from `colorPrimary` attribute instead of raw color resource since there is two themes with different primary colors. We can do it either by using [MaterialColors](https://developer.android.com/reference/com/google/android/material/color/MaterialColors?hl=en#getColor(android.view.View,%20int)) helper or directly from [theme](https://developer.android.com/reference/android/content/res/Resources.Theme#resolveAttribute(int,%20android.util.TypedValue,%20boolean)).